03 - Showcases & Templates/Vaults/Template_Hub.md

```diff
@@ -0,0 +1,49 @@
+---
+aliases: Template_Hub
+- 
```

Comment:
The minus here makes the YAML invalid. Either move the name next to the minus or remove it.